### ISSUE TITLE
Autoinduction

### ISSUE THREAD

**Issue by jgg1822** [Open-Systems-Pharmacology/PK-Sim#1284]
Is it possible to include autoinduction of CYP enzyme metabolism in PK-Sim?

**Comment by sfrechen** [comment]
Yes. If you induce an enzyme that your compound is substrate of, you will consider auto-induction.
